electron/js/preload.js

```diff
@@ -139,8 +139,10 @@ ipcRenderer.once('webapp-loaded', function(sender, config) {
   // else we are on /auth
   try {
     Object.assign(window.sodium, require('libsodium-neon'));
+    console.log('Using libsodium-neon');
   } catch (err) {
     // fallback to libsodium.js
+    console.error('Failed loading libsodium-neon', err);
```

Comment:
You can make use of the error message:
```js
console.error(`Failed loading libsodium-neon: ${err.message}`, err);
```
The vairable `err` should be also named `error`.